gateway/src/main/java/io/zeebe/gateway/impl/configuration/AIMDCfg.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.0. You may not use this file
+ * except in compliance with the Zeebe Community License 1.0.
+ */
+package io.zeebe.gateway.impl.configuration;
+
+import io.zeebe.util.DurationUtil;
+import io.zeebe.util.Environment;
+import java.time.Duration;
+
+public class AIMDCfg {
+
+  private String requestTimeout;
+  private int initialLimit = 100;
+  private int minLimit = 1;
+  private int maxLimit = 1000;
+  private double backoffRatio = 0.9;
+
+  public Duration getRequestTimeout() {
+    return DurationUtil.parse(requestTimeout);
+  }
+
+  private AIMDCfg setRequestTimeout(final String requestTimeout) {
+    this.requestTimeout = requestTimeout;
+    return this;
+  }
+
+  public int getInitialLimit() {
+    return initialLimit;
+  }
+
+  public AIMDCfg setInitialLimit(final int initialLimit) {
+    this.initialLimit = initialLimit;
+    return this;
+  }
+
+  public int getMinLimit() {
+    return minLimit;
+  }
+
+  public AIMDCfg setMinLimit(final int minLimit) {
+    this.minLimit = minLimit;
+    return this;
+  }
+
+  public int getMaxLimit() {
+    return maxLimit;
+  }
+
+  public AIMDCfg setMaxLimit(final int maxLimit) {
+    this.maxLimit = maxLimit;
+    return this;
+  }
+
+  public double getBackoffRatio() {
+    return backoffRatio;
+  }
+
+  public AIMDCfg setBackoffRatio(final double backoffRatio) {
+    this.backoffRatio = backoffRatio;
+    return this;
+  }
+
+  public void init(final Environment environment, GatewayCfg gatewayCfg) {
+    if (requestTimeout == null) {
+      setRequestTimeout(gatewayCfg.getCluster().getRequestTimeout().toMillis() + "ms");
```

Comment:
Didn't we decide on making the default half of this? e.g. `Math.floor(gatewayCfg.getCluster().getRequestTimeout().toMillis(), 2)`?

Reply:
Yes. I am not yet 100% sure what is the default value. So I left it as it is. But I will make it half.